src/Workspaces/CSharp/Portable/Formatting/CSharpFormattingOptions.cs

```diff
@@ -40,6 +40,11 @@ public static partial class CSharpFormattingOptions
                 KeyValuePairUtil.Create("no_change", LabelPositionOptions.NoIndent),
                 KeyValuePairUtil.Create("one_less_than_current", LabelPositionOptions.OneLess),
             });
+        private static readonly BidirectionalMap<string, NewLineOption> s_legacyNewLineOptionsEditorConfigMap =
```

Comment:
💡 This one does not need to be a bi-directional map. I'll file a follow-up item to change it to `ImmutableDictionary<string, NewLineOption>` (or just inline it).

Reply:
➡️ #30643